### PR TITLE
runs OpenShift specific ResourceQuota monitors

### PR DESCRIPTION
**Pull request by p0lyn0mial**
for backward compatability we need to maintain "openshift.io/imagestreams" alias for "count/imagestreams.image.openshift.io"
to dynamically update quota for imagestreams resources.

### CODE REVIEW

File: pkg/cmd/controller/quota.go [hunk 1]
```diff
@@ -46,6 +53,34 @@ func RunResourceQuotaManager(ctx *ControllerContext) (bool, error) {
...
+				return nil, err
+			}
+			for i := range resourceList.APIResources {
+				gr := schema.GroupResource{
+					Group:    gv.Group,
+					Resource: resourceList.APIResources[i].Name,
+				}
+				if evaluator := registry.Get(gr); evaluator != nil {
+					interestingResources = append(interestingResources, resourceList)
+					break
```
Comment (p0lyn0mial) [inline pkg/cmd/controller/quota.go:76]: maybe we should remove it, what if more resources in the group apply?

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Review by soltysh — APPROVED** [review]
/lgtm
/hold cancel

**Pull request merged**